### DISCUSSION TITLE
getting an error for the interger function

### DISCUSSION THREAD

**Discussion by NeilParry** [emanuelhuber/RGPR#40] [Bugs] [answered]
Just trying this package out and am not really familiar with R.

Seems to have installed ok,.. but when I run the following command:

> x <- readGPR(dsn = "LID10001.dt", verbose = TRUE)

I get

Error in interger() : could not find function "interger"

Any help appreciated

**Comment by emanuelhuber** [comment, marked as answer]
I corrected this typo. Best,
Emanuel
